docs/current/sdk/go/959738-get-started.md

```diff
@@ -75,356 +80,440 @@ func build(repoUrl string) error {
+    os.Exit(1)
+  }
+  repo := os.Args[1]
+  if err := build(repo); err != nil {
+    fmt.Println(err)
+  }
+}
+
 func build(repoUrl string) error {
   fmt.Printf("Building %s\n", repoUrl)
 
-  // 1. Get a context
+  // get context
   ctx := context.Background()
-  // 2. Initialize dagger client
+
+  // highlight-start
+  // initialize Dagger client
   client, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stdout))
   if err != nil {
     return err
   }
   defer client.Close()
-  // 3. Clone the repo using Dagger
+
+  // clone repository with Dagger
   repo := client.Git(repoUrl)
   src, err := repo.Branch("main").Tree().ID(ctx)
   if err != nil {
     return err
   }
-  // 4. Load the golang image
-  golang := client.Container().From("golang:latest")
-  // 5. Mount the cloned repo to the golang image
-  golang = golang.WithMountedDirectory("/src", src).WithWorkdir("/src")
-  // 6. Do the go build
-  _, err = golang.Exec(dagger.ContainerExecOpts{
-    Args: []string{"go", "build", "-o", "build/"},
-  }).ExitCode(ctx)
-  if err != nil {
-    return err
-  }
-  return nil
-}
-```
-
-This new code will connect to a dagger engine, clone the given git repo, load a golang container image, and build the repo.
-
-- initialize a `context.Background` for the client to use.
-- get a Dagger client with `dagger.Connect()`. This will provide the interface to execute commands against the Dagger engine.
-- clone the git repo. `client.Git()` gives a `GitRepository`, then `.Branch("main").Tree().ID()` will clone the main branch.
-- load the latest golang image with `client.Container().From("golang:latest")`.
-- mount the cloned repo with `.WithMountedDirectory("/src", src)` and set the container's working directory using `.WithWorkdir("/src")`.
-- execute the build command, `go build -o build/` by calling `Container.Exec().ExitCode()`.
-
-1. Try the `test` step of the pipeline by executing the command below from the application directory:
-
-```shell
-go build
-./multibuild https://github.com/kpenfound/greetings-api.git
-```
-
-In the output of this command, you will see Dagger cloning the git repo and running go build on it.
-
-## Step 3: Copy the build output to the host machine
-
-Once the tool has completed the build, it should put that build artifact somewhere to be used. In this step, Dagger will copy the build artifact to the host machine after the build is complete.
-
-1. Update the file `main.go` and some new steps to the `build()` function as shown below. Save the file once done.
-
-```go
-func build(repoUrl string) error {
-  ...
-  // 1. reference to the current working directory on the host
-  workdir := client.Host().Workdir() // <-- New
 
+  // get `golang` image
   golang := client.Container().From("golang:latest")
+
+  // mount cloned repository into `golang` image
   golang = golang.WithMountedDirectory("/src", src).WithWorkdir("/src")
 
-  // 2. Create the output path on the host for the build
-  // -->
+  // build application
```

Comment:
```suggestion
  // define the application build command
```

I suggest this because calling `.Exec` does not result in the build actually happening yet, it just defines the command (a common source of confusion for users, so maybe this alternative phrasing can help point them in the right direction)